package.json

```diff
@@ -1,11 +1,11 @@
 {
   "name": "eventgate",
-  "version": "1.5.1",
+  "version": "1.5.2",
```

Comment:
Technically this is a breaking change since you don't reexport the util functions.. so maybe at least 1.6.0?

Reply:
Done